code/modules/mob/thermoregulation.dm

```diff
@@ -65,7 +65,7 @@
 	if(loc_temp < bodytemperature)
 		// We're going to try and just use exposed area(temperature difference)/cold divisor, and assume we're only conducting.
 		var/thermal_loss = (1-get_cold_protection())  				// How much of your skin is exposed.
-		if(environment.total_moles > MOLES_CELLSTANDARD || !IS_SPACE_COLD)
+		if(!isVentCrawling() && (environment.total_moles > MOLES_CELLSTANDARD || !IS_SPACE_COLD))
```

Comment:
He did it again.
The absolute madman, he did the exact same thing again.
https://github.com/d3athrow/vgstation13/pull/9116